src/messages/responses/cache-list-fetch.ts

```diff
@@ -27,6 +27,10 @@ class _Hit extends Response {
     return this._values.map(v => TEXT_DECODER.decode(v));
   }
 
+  public valueList(): string[] {
```

Comment:
This would be a bigger change, but do we want to replace the word 'value' with 'element' if we're standardizing on the element language for 1.0? I don't think it reads as nicely, but we should explicitly chose whether to do it.

Reply:
I see two considerations.

1. "Hit" responses should all have the same method to get the thing from the cache, regardless of collection type, then users don't have to use differently named methods that all do the same thing depending on what sort of collection they're getting from.
2. Ideally it should be the same term as what the setters use.

You set/add an X and you get an X. Whatever X is.

Reply:
> This would be a bigger change, but do we want to replace the word 'value' with 'element' if we're standardizing on the element language for 1.0? I don't think it reads as nicely, but we should explicitly chose whether to do it.

nope, `value` has been decided and is already locked in in .NET 1.0.  Also IMO still consistent with @malandis 's terminology definitions (search for name-o-rama on Notion).

Reply:
> I see two considerations.
> 
>     1. "Hit" responses should all have the same method to get the thing from the cache, regardless of collection type, then users don't have to use differently named methods that all do the same thing depending on what sort of collection they're getting from.
> 
>     2. Ideally it should be the same term as what the setters use so it's obvious how they relate.
> 
> 
> You set/add an X and you get an X. Whatever X is.

I think this is consistent with Allen's suggestion and we will consider it for future releases.